src/apiController.js

```diff
@@ -0,0 +1,217 @@
+let axios = require('axios');
+let config = require('/src/config.js');
+
+
+axios.defaults.baseURL = config.server;
+axios.defaults.headers.common['Authorization'] = config.token;
+
+//I've basically created functions for each type of call. Didn't actually have them do anything yet because I haven't fully figured out how to use these.
+// They'll also need to have actual stuff written in them to actually use them, but I figure people can fill in the info for their own functions as we go on.
+let apiController = {
+
+
+  // Products API calls
+  getProduct: (pages, countNum) => {
+    let page = pages || 1;
+    let count = countNum || 5;
+    let params = {page: page, count: count};
+    axios.get('/products', {params})
+    .then((data) => {
+      console.log(data);
+    })
+    .catch((err) => {
+      console.error('Oh noes ', err);
+    })
+  },
+  //I'm unsure of whether I'm doing this the right way since I'm not sure how exactly the productId is going to be passed on, but I assume something like this.
+  getProductById: (productId) => {
```

Comment:
Similarly, this could become getProduct if we rename above to getProducts.

Obv dont put this parameter into an object, it's not optional, and theres only one